.ci/containers/go-ruby/Gemfile

```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+
+gem 'activesupport'
```

Comment:
This is supposed to be a copy of the Gemfile in the root directory right? And we have a copy here because this is the Gemfile baked into the container right and we want to be explicit about that fact?

Reply:
Exactly!